### PR TITLE
v0.8.0 Test 1

### PR DESCRIPTION
**Pull request by wanda-carlson**
[Screen Shot 2022-01-12 at 1 15 16 PM](https://user-images.githubusercontent.com/87515018/154271876-5b1b6d32-b3f9-4c95-93ba-23b2bed4209d.png)

This is a PR body ok **Henry Richard**?

### CODE REVIEW

**Comment by wanda-carlson** [comment]
Thanks Henry.

File: src/wanda.py [hunk 1]
```diff
@@ -1,2 +1,3 @@
 print("this is wanda")
 print('this is wanda too')
+print("Wanda Testing v0.8.0")
```
Comment (henry-richard) [inline src/wanda.py:3]: Can you make the T small?

**Review by henry-richard — APPROVED** [review]

File: src/wanda.py (hunk 1 above)
Comment (wanda-carlson) [reply src/wanda.py]: Done

**Comment by wanda-carlson** [comment]
I'll merge once its done tests

**Pull request merged**